fenix/config/detekt.yml

```
@@ -139,6 +139,7 @@ complexity:
     ignoreDefaultParameters: true # Enabled in https://github.com/mozilla-mobile/android-components/issues/10835
     ignoreDataClasses: true
     ignoreAnnotatedParameter: []
+    ignoreAnnotated: ['Composable']
```

```suggestion
    ignoreAnnotated: ['Composable'] # Configuration for Compose https://bugzilla.mozilla.org/show_bug.cgi?id=1872996
```
We just want to reference non-default changes with a comment of where the changes came from. This helps us when we need to bump detekt version and update our detekt.yml file with the base configuration changes which we pull from https://github.com/detekt/detekt/blob/main/detekt-core/src/main/resources/default-detekt-config.yml.